tests/manager/bug0357.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+PHPC-357: The exception for "invalid namespace" does not list the broken name
+--FILE--
+<?php
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
```

Comment:
Happy to add the skip-ifs, but if you mean by "constants" the URI - then no, as I needed one that is likely not to connect to anything ever. I don't have MongoDB on 44444 :-)